utils/build/virtual_machine/weblogs/nodejs/provision_test-shell-script.yml

```diff
@@ -19,7 +19,7 @@ lang_variant:
 
 weblog:
     name: test-shell-script
-    excluded_os_branches: [amazon_linux2_dotnet6, amazon_linux2023_amd64]
+    excluded_os_branches: [amazon_linux2_dotnet6, amazon_linux2023_amd64, centos_7_amd64]
```

Comment:
Is it okay for `provision_test-shell-script.yml` to remain on Node 16?

Reply:
Changed to node18